### PR TITLE
fix debug build

### PR DESCRIPTION
**Pull request by sethalves**

### CODE REVIEW

File: libraries/physics/src/MeshMassProperties.cpp [hunk 1]
```diff
@@ -283,9 +283,11 @@ void MeshMassProperties::computeMassProperties(const VectorOfPoints& points, con
     uint32_t numTriangles = triangleIndices.size() / 3;
     for (uint32_t i = 0; i < numTriangles; ++i) {
         uint32_t t = 3 * i;
+        #ifndef NDEBUG
```
Comment (ZappoMan) [inline libraries/physics/src/MeshMassProperties.cpp]: you shouldn't need this if we've got this right

**Comment by hifi-gustavo** [comment]
Build console output for commit: 9a51a4c5b18aac06d5a5c97851914e36680b217a available at: https://gist.github.com/7844d861cdab65cea56e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5152.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5152.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 44bb2201307b9e058669eb529fef4e7abc26509e available at: https://gist.github.com/7844d861cdab65cea56e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5152.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5152.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 68df6b662de3c38a36b9483c7e3bd7f65f09aefc available at: https://gist.github.com/7844d861cdab65cea56e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5152.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5152.dmg

**Pull request merged**